abhishekjakhar/portfolio
# Favicon added

### PR

#### DESCRIPTION

Added favicon for the website.


#### HOW TO TEST

- Run the app using **npm run dev**
- Open localhost:3000
- The favicon should appear
